docs/adding-and-updating-views.rst

```diff
@@ -0,0 +1,189 @@
+
+.. code-block:: sql
+
+    -- We add this with the rest of the joins in the file
+    left join
+        (select id, documents_count AS total_planning_documents from planning_summary) AS planning_summary using(id) 
+    left join
+        tmp_planning_documents_aggregates
+    using(id)
+
+4. Finally we drop the :code:`tmp_` table since we don't need it any more:
+
+.. code-block:: sql
+
+    drop table if exists tmp_planning_documents_aggregates;
+
+5. Now :ref:`test your changes<testing-changes>`.
+
+6. Then :ref:`add documentation<adding-docs>`. This is **required**!
+
+7. Run the tests to make sure your changes were successful and you didn't break anything else in the process by running: :code:`pytest`
+
+.. _testing-changes:
+
+Testing your changes
+--------------------
+
+Test your update by comparing it to the initial view you made when :ref:`loading data<loadingdata>`.
+
+  * Either create a new view: :code:`python ocdskingfisher-views-cli add-view 1 "Test: view with descriptions"`
+  * or refresh your existing view: :code:`python ocdskingfisher-views-cli refresh-views collection_1`
+  * Verify that the data is what you expect it to be.
+  * (If you're looking at the data in a postgres client, don't forget to refresh it.)
+
+.. _adding-docs:
+
+Adding documentation
+--------------------
+
+The tests won't pass if you don't document new fields!
+
+1. Add changes to the inline field-level documentation as follows:
+  * Edit the :code:`999-docs.sql` file to add the new fields and their descriptions as comments on columns. The comments should be in the same order as the tables.
+  * Eg. for the examples above:
+
+.. code-block:: sql
+
+    -- For the new field on tender:
+
+    Comment on column %%1$s.tender_id IS '`id` from `tender` object';
+    Comment on column %%1$s.tender_title IS '`title` from `tender` object';
+    Comment on column %%1$s.tender_status IS '`status` from `tender` object';
+    Comment on column %%1$s.tender_description IS '`description` from `tender` object'; -- This is our update
+    ...
+    -- For the planning document aggregates:
+
+    'Comment on column %%1$s.total_planning_documents IS ''Count of planning documents in this release''; '
+    'Comment on column %%1$s.planning_documenttype_counts IS ''JSONB object with the keys as unique planning/documents/documentType and the values as count of the appearances of those documentTypes''; '
+
+2. Test your documentation additions.
+  * Run :code:`refresh-views`, which will throw an error if you've made a typo.
+  * Preview the docs in your postgres client by looking at the schema to check the new comment appears.
+
+3. Update the CSV files of the docs by running :code:`ocdskingfisher-views-cli docs-table-ref {collection}`
```

Comment:
This should have python at the front

Reply:
One of the expectations is that people reading this will be familiar with loading data, which means I think they'll understand the schema names. I'll go through and make sure they're consistent though, and that this expectation is clearer.